docs/content/preview/migrate/reference/yb-voyager-cli.md

```diff
@@ -241,6 +410,13 @@ yb-voyager import data status --export-dir /path/to/yb/export/dir
 
 Specifies the path to the directory containing the data files to export.
 
+An export directory is a workspace used by yb-voyager to store the following:
+
+- exported schema DDL files
+- export data files
+- migration state
+- log file
+
 ### --source-db-type
```

Comment:
Why are the arguments in the Arguments section in this order? Seems random.

Reply:
I don't think order matters here because we are anyway linking them for different commands. In any case , will keep it as an improvement task.